src/Pages/kiwanlee/Login/Login.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import './Login.scss';
+//import { Link } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputId: '',
+      inputPw: '',
+      background: '#bfdffd',
+      disabled: true,
+      idChecked: false,
+      pwChecked: false,
+    };
+  }
+
+  handleInputId = e => {
+    this.setState({ inputId: e.target.value });
+    e.target.value.includes('@')
+      ? this.setState({ idChecked: true }, () => this.btnChangeColor())
+      : this.setState({ idChecked: false }, () => this.btnChangeColor());
+  };
+
+  handleInputPw = e => {
+    this.setState({ inputPw: e.target.value });
+    e.target.value.length > 4
+      ? this.setState({ inputPw: e.target.value, pwChecked: true }, () =>
+          this.btnChangeColor()
+        )
+      : this.setState({ pwChecked: false }, () => this.btnChangeColor());
+  };
```

Comment:
```suggestion
handleInputs = e => {
  this.setState({
    [e.target.name]: e.target.value
  });
}
```
이런식으로 하면 하나의 함수로 해결할 수 있습니다!!
또한 색을 변경하는 로직은 따로 리뷰 달겠습니다!! 여기서는 빼주세요